.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+* @tschaffter
+
+/apps/api/ @vpchung
```

Comment:
Nit (optional): Might be easier to do teams so you can continuously add and remove members from teams instead of editing this file

Reply:
This is a good suggestion. For now I will keep it as it is for the sake of simplicity. I will start using teams when we have more contributors and and can identify "contributor types" (e.g. API spec, API, web app).